packages/ui-extensions-server-kit/src/state/reducers/extensionServerReducer.ts

```diff
@@ -17,7 +17,7 @@ export function extensionServerReducer(state: ExtensionServerState, action: Exte
         ...state,
         store: action.payload.store,
         app: {...(state.app ?? {}), ...(action.payload.app ?? {})},
```

Comment:
Can you fix this as well? Everything should be replaced on a "connected" message

Reply:
Are you saying line 19 should just be `app`?